Makefile

```diff
@@ -5,17 +5,21 @@
 
 sagefile := .sage/bin/sagefile
 
-.PHONY: $(sagefile)
-$(sagefile):
+$(sagefile): .sage/go.mod $(shell find .sage/.. -type f -name '*.go')
 	@cd .sage && go mod tidy && go run .
 
 .PHONY: sage
+sage: clean-sage-file
 sage: $(sagefile)
 
 .PHONY: update-sage
 update-sage:
 	@cd .sage && go get -d go.einride.tech/sage@latest && go mod tidy && go run .
 
+.PHONY: clean-sage-file
+clean-sage-file:
```

Comment:
Feels like this can just be inlined in the `sage` target?

Reply:
aye

Reply:
:+1: I used one way I found in the docs of make to realize it, but I am not sure if this is the preferred way